webapp/src/utils/utils.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Source: https://www.robots.ox.ac.uk/~adutta/blog/standalone_uuid_generator_in_javascript.html
+function uuid() {
```

Comment:
There's a UUID library we could use instead of this - might be worth using that instead of re-defining it ourselves

Reply:
For sure, but I know there is a library that also does this (and probably the same way). It also would mean we wouldn't be on the hook if the implementation needed to change for security reasons (which is also a double edged sword). Just something to think about, 0/5 on actually changing it.